analysis_driver/pipelines/projects.py

```diff
@@ -17,13 +30,11 @@ def build_pipeline(dataset):
     if len(gvcf_files) < 2:
         raise PipelineError('Incorrect number of gVCF files: require at least two')
 
-    relatedness = Relatedness(
-        dataset=dataset,
-        gvcf_files=gvcf_files,
-        reference=cfg['references'][dataset.species]['fasta'],
-        project_id=dataset.name
-    )
-    output = Output(dataset=dataset, previous_stages=[relatedness])
+    reference = cfg['references'][dataset.species]['fasta']
+    genotype_gvcfs = Genotype_gVCFs(dataset=dataset, gVCFs=gvcf_files, reference=reference)
+    relatedness = Relatedness(dataset=dataset, project_id=project_id, previous_stages=[genotype_gvcfs])
```

Comment:
Since `project_id` has been removed from Relatedness, it should also be removed from this constructor call.